packages/typespec-test/test/NetworkAnalytics.Management/generated/typespec-ts/src/api/operations/index.ts

```diff
@@ -28,7 +28,7 @@ export function _listSend(
 
 export async function _listDeserialize(
   result: OperationsList200Response | OperationsListDefaultResponse,
-): Promise<PagedOperation> {
+): Promise<OperationListResult> {
```

Comment:
This change is because ARM has changed it's friendly name for operations_list result.